python/lsst/cp/pipe/ptc.py

```diff
@@ -404,10 +404,15 @@ def runDataRef(self, dataRef, visitPairs):
         allTags = []
         for (v1, v2) in visitPairs:
             # Get postISR exposures.
-            dataRef.dataId['expId'] = v1
-            exp1 = dataRef.get("postISRCCD", immediate=True)
-            dataRef.dataId['expId'] = v2
-            exp2 = dataRef.get("postISRCCD", immediate=True)
+            try:
+                dataRef.dataId['expId'] = v1
+                exp1 = dataRef.get("postISRCCD", immediate=True)
+                dataRef.dataId['expId'] = v2
+                exp2 = dataRef.get("postISRCCD", immediate=True)
+            except RuntimeError:
+                self.log.warn(f"postISR exposure for either visit {v1} or visit {v2} could not be retreived. "
+                              "Ignoring flat pair.")
+                continue
```

Comment:
Does this pair show up as masked on ~L303?  I'm trying to think if there are downstream issues due to dropping this pair.

Reply:
The skipped pair does not show up in the return of {{getVisitsUsed}} (as this function only returns those pairs that were used), so that means that it does show up as masked. Also, as you point out in Jira, the example in there runs.